layouts/partials/navigation_level.html

```diff
@@ -28,32 +28,40 @@
         {{ $pageLocation := (path.Dir (path.Dir .File.Dir)) }}
         {{ if and (eq $parentDir $pageLocation) (not .Params.draft) }}
             {{ $desc := .Description }}
+            {{ $linktitle := .LinkTitle }}
+            {{ if isset .Params "status" }}
+                {{ $status := .Params.status }}
+                {{ if in $status "experimental" }}
```

Comment:
So for experimental feature we use a dagger, what about for status: beta/experimental. I am not even sure what that status means. @craigbox any thoughts on that one?

Reply:
I think some parts of the feature are Beta and some are experimental.

I think you use the same symbol for anything and people then mouse-over to see the status.
That might lead to a discussion on if we should do that for Beta APIs too; I suspect the answer is no.